alerts/system_alerts.libsonnet

```diff
@@ -46,7 +46,7 @@
               severity: 'warning',
             },
             annotations: {
-              message: "Kubernetes API server client '{{ $labels.job }}/{{ $labels.instance }}' is experiencing {{ printf \"%0.0f\" $value }}% errors.'",
+              message: "Kubernetes API server client '{{ $labels.job_name }}/{{ $labels.instance }}' is experiencing {{ printf \"%0.0f\" $value }}% errors.'",
```

Comment:
here `job` is the correct label, as it's not the kubernetes job, but the `job` label of Prometheus